test/unit/utils/object/clear_empties.js

```diff
@@ -0,0 +1,37 @@
+const {expect} = require("chai");
+const utils = require("../../../../dist/utils");
+
+describe("utils.object.clearEmpties", () => {
+	it("Should be a function", () => {
+		expect(utils.object.clearEmpties).to.be.a("function");
+	});
+
+	const tests = [
+		{
+			"input": {},
+			"output": {}
+		},
+		{
+			"input": {"id": 1, "name": "Bob"},
+			"output": {"id": 1, "name": "Bob"}
+		},
+		{
+			"input": {"id": 1, "name": "Bob", "address": {"country": "world", "zip": 12345}},
+			"output": {"id": 1, "name": "Bob", "address": {"country": "world", "zip": 12345}}
+		},
+		{
+			"input": {"id": 1, "name": "Bob", "address": {}},
+			"output": {"id": 1, "name": "Bob"}
+		},
+		{
+			"input": {"id": 1, "name": "Bob", "address": {"country": "world", "zip": 12345, "other": {}}},
+			"output": {"id": 1, "name": "Bob", "address": {"country": "world", "zip": 12345}}
+		}
+	];
```

Comment:
Would love to see some tests for empty arrays here.

Reply:
Added.

Reply:
Done.